tutorials/tour/_posts/2017-02-13-operators.md

```diff
@@ -30,9 +53,26 @@ def xor(x: MyBool, y: MyBool) = (x or y) and not(x and y)
 
 This helps to make the definition of `xor` more readable.
 
-Here is the corresponding code in a more traditional object-oriented programming language syntax:
-
-```tut
-def not(x: MyBool) = x.negate
-def xor(x: MyBool, y: MyBool) = x.or(y).and(x.and(y).negate)
+## Precedence
+When an expression uses multiple operators, the operators are evaluated based on the priority of the first character:
+```
+(characters not shown below)
+* / %
++ -
+:
+= !
+< >
+&
+^
+|
+(all letters)
+```
+This applies to functions you define. For example, the following expression:
+```
+a + b ^? c ?^ d less a ==> b | c
+```
+Is equivalent to
+```
+((a + b) ^? (c ?^ d)) less ((a ==> b) | c)
 ```
+`?^` has the highest precedence because it starts with the character `?`. `+` has the second highest precedence, followed by `^?`, `==>`, `|`, and `less`.
```

Comment:
we are missing the suffix `:`

for example:

```
case class Foo(v: Int) {
  def bar:(x: Int): = Foo(v + x)
}

1 bar: Foo(2)
```

is equivalent to `Foo(2).bar:(1)`

a notable example is `val a = 1 :: 2 :: 3 :: Nil`